irksome/stepper.py

```diff
@@ -38,7 +39,8 @@ class TimeStepper:
 
     """
     def __init__(self, F, butcher_tableau, t, dt, u0, bcs=None,
-                 solver_parameters=None, bc_type="DAE"):
+                 solver_parameters=None, bc_type="DAE", splitting=AI):
+        self.splitting = splitting
```

Comment:
`splitting` is never needed outside `__init__` so could just not save it.

Reply:
Correct,  not sure if we'll ever need it or not, but it's easy to put back if we do.